### PR TITLE
etcd: import e2e 386 and arm64 presubmit jobs

### PR DESCRIPTION
**Pull request by ivanvc**
Imported the following e2e jobs:

* [386](https://github.com/etcd-io/etcd/blob/407eaa049dbd92c3c6a290e4df90e8e349a1157c/.github/workflows/e2e.yaml#L33-L35)
* [arm64](https://github.com/etcd-io/etcd/blob/407eaa049dbd92c3c6a290e4df90e8e349a1157c/.github/workflows/e2e-arm64.yaml#L30-L32)

Link to #32754 / https://github.com/kubernetes/k8s.io/issues/6102

### CODE REVIEW

**Review by jmhbnz — COMMENTED** [review]
/lgtm

Thanks @ivanvc, one step closer to completing GitHub actions migration 🎉

File: config/jobs/etcd/etcd-presubmits.yaml [hunk 1]
```diff
@@ -205,6 +205,66 @@ presubmits:
...
+        resources:
+          requests:
+            cpu: "4"
+            memory: "8Gi"
+          limits:
+            cpu: "4"
+            memory: "8Gi"
+
+  - name: pull-etcd-e2e-arm64
+    cluster: eks-prow-build-cluster
```
Comment (ivanvc) [inline config/jobs/etcd/etcd-presubmits.yaml:238]: @jmhbnz, this should rather be the `k8s-infra-prow-build` cluster, right? I just realized about this as I was doing other edits to the file.

Comment (jmhbnz) [reply config/jobs/etcd/etcd-presubmits.yaml:238]: Yes good catch, refer: https://github.com/kubernetes/test-infra/pull/33224

> They are only available on k8s-infra-prow-build cluster(for now, we'll enable the eks-prow-build-cluster in the future).

Comment (ivanvc) [reply config/jobs/etcd/etcd-presubmits.yaml]: Done, updated it.

**Comment by ahrtr** [comment]
Thanks @ivanvc 

/lgtm
/approve

/hold

@jmhbnz Please feel free to unhold this PR.

**Pull request merged**